kubeflow-pipelines/odh-ray-cluster/secret-generator.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: viaduct.ai/v1
+kind: ksops
+metadata:
+  # Specify a name
+  name: example-secret-generator
```

Comment:
it doesn't _really_ matter, but probably should name this something other than 'example-'...